res/i18n/fr/labels.json

```diff
@@ -481,38 +481,38 @@
 		"enabled": "Activé",
 		"fade_in": "Fondu ouverture",
 		"fade_out": "Fondu fermeture",
-		"head_cut": "Écrêtage",
+		"head_cut": "Coupe du début",
 		"listen": "Écoute",
-		"makeup": "Surgain",
+		"makeup": "Gain",
 		"predelay": "Pré-délai",
 		"reverse": "Inverser",
 		"sample_num": "Échantillon #",
-		"tail_cut": "Équeutage"
+		"tail_cut": "Coupe de la fin"
 	},
 	"settings": "Paramètres",
 	
-	"side_direction": "Direction de côté", 
+	"side_direction": "Cotés directivité", 
 	"sidechain": "Chaîne latérale",
 	
 	"signal": {
 		"M_iddle": "M",
 		"P_hase": "P",
 		"S_ide": "C",
 		"amplitude": "Amplitude",
-		"clip": "Clip",
-		"clipping": "Clipping",
+		"clip": "Saturation",
+		"clipping": "Saturation",
 		"dry": "Originel",
 		"dry:db": "Originel\n(dB)",
 		"dry_pan": "Pan originel",
-		"duty_ratio": "Ratio obligatoire",
+		"duty_ratio": "Rapport cyclique",
```

Comment:
I'm unsure about what this is/does, but this proposal sounds wrong to me.
@sadko4u : could you please enlighten us about what this is/does?

Reply:
@sadko4u : ping

Reply:
It's https://fr.wikipedia.org/wiki/Rapport_cyclique